### DISCUSSION TITLE
Open query for Logs Explorer

### DISCUSSION THREAD

**Discussion by Magnuti** [supabase/supabase#22447] [Questions] [answered]
Is it possible to query Logs Explorer without the Supabase dashboard?

I see that the dashboard sends a request to this endpoint `https://api.supabase.com/platform/projects/<project_id>/analytics/endpoints/logs.all?project=<project_id>&iso_timestamp_start=<timestamp>&sql=<SQL query>`

However I am lost on the bearer token used here. Is it possible to mint your own bearer token for this endpoint from the JWT secret? We cannot use the service_role key here no?

I am asking because I want to build a custom dashboard with data from Logs Explorer, and for that I need to authenticate with Supabase.

**Comment by TheOtherBrian1** [comment, marked as answer]
The [SQL Editor](https://supabase.com/dashboard/project/_/sql/new) in the Dashboard is used to query your own database. The [Query Logs Explorer](https://supabase.com/dashboard/project/_/logs/explorer) [houses data externally](https://supabase.com/blog/supabase-acquires-logflare), unless you self-host. As of now, data from the Logs Editor can only be queried within the Dashboard. 

There is a [metrics endpoint ](https://supabase.com/docs/guides/platform/metrics) for consuming data about your database, such as real-time memory and CPU usage, active direct connects, and database size. It's used mainly with the [Supabase Grafana Repo](https://github.com/supabase/supabase-grafana/blob/main/docs/supabase-grafana-prometheus.png), which can be set up to monitor your database metrics.
